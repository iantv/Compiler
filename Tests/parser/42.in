typedef int *ptr2int;
ptr2int a;
void main(){
}