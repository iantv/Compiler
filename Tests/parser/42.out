main: function
	{
	}
returns
	void
a: pointer(s) to
	int
ptr2int: alias for
	pointer(s) to
		int
